File: count zero.py
def count_zeros(matrix):
    return sum(1 for row in matrix for val in row if val == 0)

# Example
matrix = [
    [0, 2, 3],
    [4, 0, 6],
    [0, 8, 0]
]

print("Number of zeros:", count_zeros(matrix))

File: count zero.c
#include <stdio.h>

int main() {
    int rows, cols, count = 0;
    printf("Enter rows and columns: ");
    scanf("%d %d", &rows, &cols);

    int mat[rows][cols];

    printf("Enter matrix elements:\n");
    for (int i = 0; i < rows; i++) {
        for (int j = 0; j < cols; j++) {
            scanf("%d", &mat[i][j]);
            if (mat[i][j] == 0)
                count++;
        }
    }

    printf("Number of zeros: %d\n", count);

    return 0;
}
